### PR TITLE
Update create flow for pvc, routes, storage classes.

### PR DESCRIPTION
**Pull request by jcaianirh**
PR for jira card: https://jira.coreos.com/browse/CONSOLE-1047

Update create flow for pvc, routes, storage classes: to default to form editor:

- Add Edit YAML Link option in upper right corner. 

- Eliminate dropdown create buttons for these workloads. 

- OnCancel of YAML editor, route the user back to the list page.

### CODE REVIEW

File: frontend/public/components/storage-class-form.tsx [hunk 1]
```diff
@@ -795,8 +796,14 @@ export class StorageClassForm_ extends React.Component<StorageClassFormProps, St
...
       <div className="co-m-pane__body">
         <Form className="wizard-form">
-          <h1 className="co-m-pane__heading">Create Storage Class</h1>
-
+          <h1 className="co-m-pane__heading co-m-pane__heading--baseline">
+            <div className="co-m-pane__name">
+              Create Storage Class
+            </div>
+            <div className="co-m-pane__heading-link">
+              <Link to="/k8s/cluster/storageclasses/new?backCount=2">Edit YAML</Link>
```
[the remark this answers is not in the document]
Comment (jcaianirh) [reply frontend/public/components/storage-class-form.tsx]: @spadgett yes much simpler!

File: frontend/public/components/edit-yaml.jsx [hunk 2]
```diff
@@ -72,6 +70,15 @@ export const EditYAML = connect(stateToProps)(
       }
     }
 
+    onCancel() {
+      if ('onCancel' in this.props) {
+        return this.props.onCancel;
```
[the remark this answers is not in the document]
Comment (jcaianirh) [reply frontend/public/components/edit-yaml.jsx]: @spadgett no longer relevent :)

**Comment by spadgett** [comment]
Looks good. We'll have to figure out how to handle this in the tests.

**Comment by spadgett** [comment]
lgtm, please squash

**Pull request merged**